go/kbfs/libkbfs/keybase_daemon_local.go

```diff
@@ -890,14 +890,23 @@ func (k *KeybaseDaemonLocal) FavoriteDelete(
 
 // FavoriteList implements KeybaseDaemon for KeybaseDaemonLocal.
 func (k *KeybaseDaemonLocal) FavoriteList(
-	ctx context.Context, sessionID int) ([]keybase1.Folder, error) {
+	ctx context.Context, sessionID int) (keybase1.FavoritesResult, error) {
 	if err := checkContext(ctx); err != nil {
-		return nil, err
+		return keybase1.FavoritesResult{}, err
 	}
 
 	k.lock.Lock()
 	defer k.lock.Unlock()
-	return k.favoriteStore.FavoriteList(k.currentUID)
+	// TODO: determine if this is okay
```

Comment:
This is fine with me.  This is only used for testing really, so unless we write a test that relies on ignored/new folders, we should be ok here.